src/custom/components/TransactionSettings/TransactionSettingsMod.tsx

```diff
@@ -13,10 +13,13 @@ import { ThemedText } from 'theme'
 import { AutoColumn } from 'components/Column'
 import QuestionHelper from '../QuestionHelper'
 import { RowBetween, RowFixed } from 'components/Row'
+import ReactGA from 'react-ga4'
 
 // MOD imports
 import { INPUT_OUTPUT_EXPLANATION, MINIMUM_ORDER_VALID_TO_TIME_SECONDS } from 'constants/index'
 
+const MAX_DEADLINE_MINUTES = 180 // 3h
```

Comment:
This changes now right?

Reply:
based on https://cowservices.slack.com/archives/C0361CDD1FZ/p1654183473951349?thread_ts=1654139901.215619&cid=C0361CDD1FZ but checking again i think it's A-OK

Reply:
resolving, all good